Add stub page for Quilt & Nextflow (#3136)
quiltdata/quilt: docs/examples/nextflow.md

<!-- markdownlint-disable -->
# Nextflow nf-quilt3

> ##  Beta preview

Reviewer: ```suggestion
##  Technology preview
```

> `nf-quilt` is changing rapidly and therefore is not yet recommended for production work.

Reviewer: ```suggestion
`nf-quilt3` is evolving rapidly and therefore is not yet recommended for production work.
```

> `nf-quilt3` is evolving rapidly and therefore is not yet recommended for production work.

Reviewer: ```suggestion
`nf-quilt3` is evolving rapidly and therefore is not yet recommended for production work.
You may encounter bugs and missing functionality.
```

<!-- markdownlint-disable -->
# Nextflow nf-quilt3

##  Beta preview
`nf-quilt3` is evolving rapidly and therefore is not yet recommended for production work.

## What is Nextflow?

[Nextflow](https://nextflow.io) is free, open source software
distributed under the Apache 2.0 licence developed by [Seqera
Labs](http://www.seqera.io/). The software is used by scientists
and engineers to write, deploy and share data-intensive, highly
scalable, workflows on any infrastructure.

## Overview

> Nextflow plugin for interacting with [Quilt](https://quiltdata.com/) packages as a FileSystem.

Reviewer: ```suggestion
Nextflow plugin for interacting with [quilt packages](https://github.com/quiltdata/quilt) as a FileSystem.
```
